pages/[league]/player/[id].tsx

```diff
@@ -117,39 +180,77 @@ export default ({ playerId, league }: { playerId: string; league: League }) => {
           title: playerNameInfo?.name ?? 'Player',
         }}
       />
-      <Header league={league} activePage="players" />
-      <div className="mx-auto w-full bg-grey100 p-[2.5%] lg:w-3/4 lg:px-0 lg:pt-px lg:pb-10">
+      {isIndexView && <Header league={league} activePage="players" />}
+      <div
+        className={classnames(
+          'mx-auto w-full bg-grey100',
+          isIndexView && 'p-[2.5%] lg:w-3/4 lg:px-0 lg:pt-px lg:pb-10',
+        )}
+      >
         {!playerInfo || !playerRatings || !playerStats ? (
           <div className="flex h-full w-full items-center justify-center">
-            <Spinner size="xl" />
+            <Spinner id="player-loading" size="xl" />
           </div>
         ) : (
-          <div className="mx-auto lg:w-11/12">
+          <div
+            id="player-details"
+            className={classnames('mx-auto', isIndexView && 'lg:w-11/12')}
+          >
             <div className="flex flex-col items-center md:mr-8 md:flex-row md:justify-end">
-              <SeasonTypeSelector className="top-7 !h-7 w-48" />
+            <SeasonTypeSelector className={`${isIndexView ? 'top-7 !h-7 w-48' : ''} mb-4 md:mb-0 md:ml-4`} />
```

Comment:
We should be using the `classnames` package in these situations. It does a much better job coalescing these and lets us avoid this pattern.
```suggestion
            <SeasonTypeSelector className={classnames("mb-4 md:mb-0 md:ml-4", isIndexView && 'top-7 !h-7 w-48'})} />
```